working-groups.md

```diff
@@ -122,9 +122,10 @@ This is the [`tektoncd/cli`](https://github.com/tektoncd/cli) working group.
 
 | &nbsp;                                                        | Facilitators      | Company | Profile                                           |
 | --------------------------------------------------------      | ----------        | ------- | ---------------------------------------           |
-| <img width="30px" src="https://github.com/vdemeester.png">    | Vincent Demeester | Red Hat | [vdemeester](https://github.com/vdemeester)       |
 | <img width="30px" src="https://github.com/chmouel.png">       | Chmouel Boudjnah  | Red Hat | [chmouel](https://github.com/chmouel)             |
 | <img width="30px" src="https://github.com/danielhelfand.png"> | Daniel Helfand    | VMware  | [danielhelfand](https://github.com/danielhelfand) |
+| <img width="30px" src="https://github.com/pradeepitm12.png"   | Pradeep Kumar     | Red Hat | [pradeepitm12](https://github.com/pradeepitm12)   |
```

Comment:
i think there's a slight error here:

![image](https://user-images.githubusercontent.com/432502/91194278-016c2880-e6c6-11ea-8baa-95c06482f264.png)

Reply:
ah yeah it's missing a `>`..